test/functional/p2p_node_network_limited.py

```diff
@@ -35,7 +35,7 @@ class NodeNetworkLimitedTest(BitcoinTestFramework):
     def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 3
-        self.extra_args = [['-prune=2200', '-addrmantest'], [], []]
+        self.extra_args = [['-prune=2200', '-peerblockfilters=0', '-blockfilterindex=0', '-addrmantest'], [], []]
```

Comment:
Since prune is on doesn't this turn off block filters already?

Reply:
I made those test fixes before https://github.com/litecoin-project/litecoin/pull/988/commits/0d04e754aac48c94df0296b17db056725a982833. I guess I can revert them now